content/en/internal_developer_portal/software_catalog/entity_model/ai-generated-systems.md

```diff
@@ -0,0 +1,68 @@
+---
+title: AI-generated Systems
+description: Discover recommended System entities powered by Bits AI to organize your Software Catalog.
+disable_toc: false
+further_reading:
+- link: "/internal_developer_portal/software_catalog/entity_model/entity_types"
+  tag: "Documentation"
+  text: "Entity Types"
+- link: "/internal_developer_portal/software_catalog/set_up/discover_entities"
+  tag: "Documentation"
+  text: "Discover Entities"
+---
+
+## Overview
+
+AI-generated Systems surface recommended System entities that you can add to your Software Catalog. Bits AI generates these recommendations using:
+- Observability data, including distributed traces
+- Component dependencies (automatically detected with APM or manually declared)
+- Ownership metadata from your Software Catalog
+
+Recommendations include service, datastore, and queue entity types and automatically populate the following System metadata:
+- Name and display name
+- Owner
+- Description
+- Components
+
+**Note**: AI-generated Systems only suggest new Systems; they do not propose updates to existing ones.
+
+## Enable AI-generated Systems
+
+AI-generated Systems must be enabled once per organization by a user with the `org_management` permission (included in the default [Datadog Admin role][1]).
+
+To enable, navigate to **IDP** > **Software Catalog** > [**Suggestions**][2], and click **Enable System Recommendations**.
+
+{{< img src="/tracing/internal_developer_portal/enable-system-recs.png" alt="The Suggestions page in Software Catalog, showing an option to enable System Recommendations" style="width:100%;" >}}
+
+Recommendations may take up to 30 minutes to appear after enablement.
```

Comment:
Maybe consider putting this above the image? Otherwise, I think it might be easy to miss the single line of text below it.